lang/c.md

```diff
@@ -61,7 +74,7 @@ sudo apt install gcc
 sudo apt install build-essential
 ```
 
-#### Windows
+## 🪟 *Windows*
```

Comment:
Maybe use this emoji instead? For me, it cannot be seen at GitHub. 👀
```suggestion
## 💻 *Windows*
```

Reply:
![image](https://user-images.githubusercontent.com/75872316/136088185-d7320339-9efc-4bd1-bd54-4287b3bfa370.png)

It seems to be likewise!!

Reply:
Updated it to look bold now. 🙂